app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,15 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <color name="background">@color/white</color>
+    <color name="background_dark">#160039</color>
+    <color name="bubble_sent">#DAC2FF</color>
+    <color name="bubble_received">#F1F1F1</color>
+    <color name="gray">#C7C7C7</color>
+    <color name="light_gray">#74C7C7C7</color>
+    <color name="message">#191919</color>
+    <color name="primaryColor">#9F60FF</color>
+    <color name="divider">#F7CE7F</color>
```

Comment:
try naming the colors by their names, with these names I can't tell what color it is (bubble_sent,bubble_recieved and etc.)

Reply:
also Camel case for color names is bad practice

Reply:
![image](https://user-images.githubusercontent.com/95241918/232745734-b3007b3a-86ee-40b2-a896-719fb7d96f6b.png)